.github/workflows/binary.yml

```diff
@@ -7,17 +7,20 @@ on:
   schedule:
     # Run every morning to detect flakiness and broken dependencies
     - cron: '17 8 * * *'
+env:
+  ROS_DISTRO: galactic
+  BRANCH: master
```

Comment:
Where ABI stability is desired are:

foxy branch for the foxy distro
galactic branch for the galactic distro

the master branch targets the rolling distro where we have absolutely no intentions in keeping ABI stability

Reply:
*actually* we need "current branch" to whatever the target is

Reply:
Have we already branched for galactic?

I was thinking that I would just submit a new pull request for each branch to specify the target branch once this one has been approved.

Reply:
yep, branched for galactic over a month ago